File: 프로그래머스/0/120921. 문자열 밀기/문자열 밀기.js
function solution(A, B) {
    
    
    for(let i=0;i<A.length;i++){
        const shiftA = A.slice(A.length-i) + A.slice(0,A.length-i)
        console.log(shiftA)
        if(shiftA === B){
            return i
        }
    }
    return -1;
}

File: 프로그래머스/0/120921. 문자열 밀기/문자열 밀기.kt
class Solution {
    fun solution(A: String, B: String): Int {
        var answer: Int = 0
        var A_len = A.length
        
        for(i in 0 until A_len){
            var A_sub = A.substring(A_len-i)+A.substring(0,A_len-i)
            println(A_sub)
            if(A_sub==B){
                return i
            }
        }
        return -1
    }
}